### PR TITLE
utils/build_ds_container.sh: Add option to push to any image repository

### PR DESCRIPTION
**Pull request by JAORMX**
This adds the option to push to any given repository with the
aforementioned utility. This is useful for when developing content for
Kubernetes distros other than OpenShift.

A Dockerfile to take this into use was also created.

Co-Authored-By: Vincent Shen <wenshen@redhat.com>
Co-Authored-By: Lance Bragstad <lbragsta@redhat.com>
Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

File: utils/build_ds_container.sh [hunk 1]
```diff
@@ -72,7 +82,39 @@ if [ "$namespace" == "openshift-compliance" ]; then
...
+        # This enables us to create a configuration so we can build a container
+        # with the datastream
+        # If they already exist, this is not a problem
+        oc apply -n "$namespace" -f "$root_dir/ocp-resources/ds-from-local-build.yaml"
+
+        # Create output directory
+        from_dir=$(mktemp -d)
+
+        # Copy datastream files to output directory
+        cp "$root_dir/build/"*-ds.xml "$from_dir"
```
Comment (Vincent056) [inline utils/build_ds_container.sh:106]: Do we still need those, since we are just copying from the build folder in Dockerfile?

**Review by Vincent056 — COMMENTED** [review]
Just some nits.

File: utils/build_ds_container.sh (hunk 1 above)
Comment (JAORMX) [reply utils/build_ds_container.sh:106]: we do, this isn't for the registry push, but for the OpenShift push instead

Comment (Vincent056) [reply utils/build_ds_container.sh:106]: Ahh, I think it is still used here https://github.com/ComplianceAsCode/content/blob/7b64c1e178e8d820894ccf7ba55fcb8f962716f5/utils/build_ds_container.sh#L161

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Pull request merged**